deploy/helm/elastic-agent/values.schema.json

```diff
@@ -331,6 +331,48 @@
                 }
             }
         },
+        "autoOps": {
+            "type": "object",
+            "description": "Configuration for AutoOps integration.",
+            "properties": {
+                "enabled": {
+                    "type": "boolean",
+                    "description": "Enable AutoOps integration."
+                },
+                "autoops_token": {
+                    "type": "string",
+                    "description": "Token to register into AutOps"
+                },
+                "autoops_otel_url": {
+                    "type": "string",
+                    "description": "Otel server URL"
+                },
+                "autoops_temp_resource_id": {
+                    "type": "string",
+                    "description": "..."
+                },
+                "es_password": {
+                    "type": "string",
+                    "description": "Monitored ES cluster password"
+                },
+                "es_username": {
+                    "type": "string",
+                    "description": "Monitored ES cluster username"
+                },
+                "es_api_key": {
+                    "type": "string",
+                    "description": "Monitored ES cluster api key"
+                },
+                "elastic_cloud_connected_mode_api_key": {
+                    "type": "string",
+                    "description": "Cloud connected api key"
+                },
+                "elastic_cloud_connected_mode_api_url": {
```

Comment:
we need also some validations here when the autoOps.enabled is set true I expect several of these properties not be empty

Reply:
Hey, sorry, but I'm not sure how you want it to look. Could you please try to explain it better, or perhaps send an example?